### PR TITLE
UCT/TCP: fix compilation in cm_open

### PR DESCRIPTION
**Pull request by alinask**

### CODE REVIEW

File: src/uct/tcp/tcp_md.c [hunk 1]
```diff
@@ -80,7 +80,12 @@ uct_component_t uct_tcp_component = {
     .rkey_release       = ucs_empty_function_return_success,
     .name               = UCT_TCP_NAME,
     .md_config          = UCT_MD_DEFAULT_CONFIG_INITIALIZER,
-    .cm_config          = UCS_CONFIG_EMPTY_GLOBAL_LIST_ENTRY,
+    .cm_config          = {
+        .name           = "TCP-SOCKCM connection manager",
+        .prefix         = "TCP_",
+        .table          = uct_cm_config_table,
+        .size           = sizeof(uct_cm_config_t),
```
Comment (dmitrygx) [inline src/uct/tcp/tcp_md.c]: no need for trailing `,` at 87 line

Comment (alinask) [reply src/uct/tcp/tcp_md.c]: @dmitrygx we have this comma in all other similar places so i kept here the same:
https://github.com/openucx/ucx/blob/master/src/uct/ib/base/ib_md.c#L1647
https://github.com/openucx/ucx/blob/master/src/uct/sm/mm/base/mm_md.h#L168
https://github.com/openucx/ucx/blob/master/src/uct/sm/knem/knem_md.c#L395
and in all other table configs...

Comment (dmitrygx) [reply src/uct/tcp/tcp_md.c]: np then
;)

**Comment by mellanox-github** [comment]
<details><summary>Mellanox CI: <b>PASSED</b#  > on 25 workers (click for details)</summary>

**Note:** the logs will be deleted after 18-Dec-2019

Agent/Stage | Status
-- | --
[_main](https://gist.githubusercontent.com/mellanox-github/148c28e21d8437d0773771e8513bc13c/raw/62db92fec074c0479f4868b78bde068cdbdc34a3/_main) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/148c28e21d8437d0773771e8513bc13c/raw/ea729896137f9637451b21c2a6041d98742ab232/hpc-arm-cavium-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/148c28e21d8437d0773771e8513bc13c/raw/938db0f69203c288c88c08575dd1503ba58e6cac/hpc-arm-cavium-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/148c28e21d8437d0773771e8513bc13c/raw/b263de58043bef0f9a108376fa744f7a637ad553/hpc-arm-cavium-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-cavium-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/148c28e21d8437d0773771e8513bc13c/raw/1de7a5736ba700be9051bb782537d3cefbecf5c9/hpc-arm-cavium-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/148c28e21d8437d0773771e8513bc13c/raw/cfb98aba2d0b290334e4c4babbbaf5ef220e9beb/hpc-arm-hwi-jenkins_W0) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/148c28e21d8437d0773771e8513bc13c/raw/8081b2ad4b95a200962bc636dcb518dbd8c70dca/hpc-arm-hwi-jenkins_W1) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/148c28e21d8437d0773771e8513bc13c/raw/04f7a0dadf58141c4c0dd1207c8dbee3095dfe4d/hpc-arm-hwi-jenkins_W2) | :heavy_check_mark: SUCCESS
[hpc-arm-hwi-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/148c28e21d8437d0773771e8513bc13c/raw/421ec786e1899366a0413531f54491c9afe9dbba/hpc-arm-hwi-jenkins_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W0](https://gist.githubusercontent.com/mellanox-github/148c28e21d8437d0773771e8513bc13c/raw/25897fdb5dd13d6898e5878e1aa4e5e210e94588/hpc-test-node-gpu_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W1](https://gist.githubusercontent.com/mellanox-github/148c28e21d8437d0773771e8513bc13c/raw/18b63e2c60aacceb344b66c895d021f36a51a32e/hpc-test-node-gpu_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W2](https://gist.githubusercontent.com/mellanox-github/148c28e21d8437d0773771e8513bc13c/raw/4bac4362a7a30d7482a9971e4d55d3a3732a37d7/hpc-test-node-gpu_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-gpu_W3](https://gist.githubusercontent.com/mellanox-github/148c28e21d8437d0773771e8513bc13c/raw/946fdd852d499b09ea3eeb63e238693b5f4a3fe3/hpc-test-node-gpu_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W0](https://gist.githubusercontent.com/mellanox-github/148c28e21d8437d0773771e8513bc13c/raw/8b3102d12fb3a6260d0cb69a9bccd8bb63a5b72d/hpc-test-node-legacy_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W1](https://gist.githubusercontent.com/mellanox-github/148c28e21d8437d0773771e8513bc13c/raw/81d74267e44e54b8bf86c771a8b4667aa67dbbee/hpc-test-node-legacy_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W2](https://gist.githubusercontent.com/mellanox-github/148c28e21d8437d0773771e8513bc13c/raw/3b4519fb52a0996db8c6c569b9d2ba240d425c04/hpc-test-node-legacy_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-legacy_W3](https://gist.githubusercontent.com/mellanox-github/148c28e21d8437d0773771e8513bc13c/raw/a7cb1485cc42441b6ba3711b4426e5d860d65a08/hpc-test-node-legacy_W3) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W0](https://gist.githubusercontent.com/mellanox-github/148c28e21d8437d0773771e8513bc13c/raw/2aa29a365eaca83cc8435662817db6aa578c33f0/hpc-test-node-new_W0) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W1](https://gist.githubusercontent.com/mellanox-github/148c28e21d8437d0773771e8513bc13c/raw/baca12ce30dae0de820e3c5b8b21a8288db7da82/hpc-test-node-new_W1) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W2](https://gist.githubusercontent.com/mellanox-github/148c28e21d8437d0773771e8513bc13c/raw/9c565b071290ed856c0a69889ac4c59f5f266b3b/hpc-test-node-new_W2) | :heavy_check_mark: SUCCESS
[hpc-test-node-new_W3](https://gist.githubusercontent.com/mellanox-github/148c28e21d8437d0773771e8513bc13c/raw/757ca67166fc15225fb2664818c8daa8af01a3fc/hpc-test-node-new_W3) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W0](https://gist.githubusercontent.com/mellanox-github/148c28e21d8437d0773771e8513bc13c/raw/b68f62283dadd97b53232712573e03ceafa8883c/r-vmb-ppc-jenkins_W0) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W1](https://gist.githubusercontent.com/mellanox-github/148c28e21d8437d0773771e8513bc13c/raw/6919c17f4c65ae8ec061b8251b279eecedbe6c41/r-vmb-ppc-jenkins_W1) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W2](https://gist.githubusercontent.com/mellanox-github/148c28e21d8437d0773771e8513bc13c/raw/16c8afe90f4f1855616c5ac8acb6c9415c97a812/r-vmb-ppc-jenkins_W2) | :heavy_check_mark: SUCCESS
[r-vmb-ppc-jenkins_W3](https://gist.githubusercontent.com/mellanox-github/148c28e21d8437d0773771e8513bc13c/raw/c4fd2cc0a1539460983040ebf572abfab550352d/r-vmb-ppc-jenkins_W3) | :heavy_check_mark: SUCCESS

**Comment by alinask** [comment]
This PR fixes a compilation error that was caused by merging 2 PRs - first one added a tcp cm and the second changed the API that opens a cm.
Merging this now to fix the master branch.
@yosefe

**Pull request merged**